src/components/phone/CallerIDEditCard.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div>
+    <base-button
+      variant="outline"
+      :action="
+        () => {
+          isShowingModal = true;
+        }
+      "
+      text="Edit Caller ID"
+      class="px-3 py-2"
+    ></base-button>
+    <modal
+      v-if="isShowingModal"
+      modal-classes="w-108"
+      @ok="isShowingModal = false"
+      @close="isShowingModal = false"
+    >
+      <div class="flex flex-col justify-around ml-12 mr-12 mt-8">
+        <!-- Greeting -->
+        <base-text
+          variant="body"
+          weight="700"
+          class="text-crisiscleanup-dark-500 text-center pb-3"
+        >
+          Welcome back John!
+        </base-text>
+        <base-text
+          variant="body"
+          weight="300"
+          class="text-crisiscleanup-dark-300 text-center pb-3"
+        >
+          Please confirm the number you would like to use today. Don't worry, we
+          will hide it on all inbound and outbond calls for your protection
+        </base-text>
+        <div class="flex flex-col justify-around">
+          <!-- Phone # -->
+          <base-button
+            variant="outline"
+            size="medium"
+            :ccu-icon="togglePhone ? 'updown' : 'up'"
+            :action="() => (togglePhone = !togglePhone)"
```

Comment:
boom. that's correct, seeing this then I'm assuming the above thing was just a copy/paste error :P